src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1800,24 +1800,27 @@ private void OnEditorReady()
 
         gui.UpdateTimeIndicator(CurrentGame.GameWorld.TotalPassedTime);
 
-        // Get summary before applying results in order to get comparisons to the previous populations
-        var run = CurrentGame.GameWorld.GetAutoEvoRun();
-
-        if (run?.Results == null)
-        {
-            gui.UpdateAutoEvoResults("Auto-evo failed to run", "run status: " +
-                (run != null ? run.Status : string.Empty));
-        }
-        else
+        if (Settings.Instance.RunAutoEvoDuringGamePlay.Value)
```

Comment:
Isn't this like the completely wrong way to do this? Now auto-evo is completely disabled if the player disables the option to run auto-evo during gameplay. Instead the auto-evo run needs to be started when entering the editor.